### PR TITLE
docs:[ produce] replace two broken link

### PR DESCRIPTION
**Pull request by shivam-sharma7**
Signed-off-by: Shivam Sharma <meshivam81@gmail.com>

### Related Issue

Solving Issue: #209 

### PR Summary
 Replace two links in the `Send an event with delay `  

### PR Type

- [ ] Fix typos or format(punctuation, space, indentation, code block, etc.)
- [ ] Update inappropriate or misunderstanding content
- [x] Add missing content or new documents
- [ ] Delete useless content or outdated documents
- [ ] Localize or translate documents
- [ ] Deal with GitHub action files and scripts
- [ ] Deal with chores

### CODE REVIEW

**Comment by JieDing** [comment]
@shivam-sharma7 thanks for your contribution, can you update the link: 
image

**Comment by shivam-sharma7** [comment]
> @shivam-sharma7 thanks for your contribution, can you update the link: image

review the PR / @JieDing

File: content/how-to/produce/send-delay-event.md [hunk 1]
```diff
@@ -11,8 +11,8 @@ The following document will teach you how to build a simple producer with a time
 **Prerequisites**
 
 To send events to Vanus, you must meet the following prerequisites:
-1. Have a running [Vanus](https://github.com/linkall-labs/docs/blob/main/user-manual/getting-started/install/k8s(recommended).md) cluster.
-2. Have [vsctl](https://github.com/linkall-labs/docs/blob/main/user-manual/how-to/vsctl.md).
+1. Have a running [Vanus](https://github.com/linkall-labs/docs/blob/main/content/getting-started/installation.mdx) cluster.
```
Comment (JieDing) [inline content/how-to/produce/send-delay-event.md:14]:
```suggestion
1. Have a running [Vanus](/getting-started/installation) cluster.
```

File: content/how-to/produce/send-delay-event.md [hunk 2]
```diff
@@ -11,8 +11,8 @@ The following document will teach you how to build a simple producer with a time
...
+2. Have [vsctl](https://github.com/linkall-labs/docs/blob/main/content/how-to/vsctl.md).
```
Comment (JieDing) [inline content/how-to/produce/send-delay-event.md:15]:
```suggestion
2. Have [vsctl](/how-to/vsctl).
```

**Review by JieDing — COMMENTED** [review]
@shivam-sharma7 See comments

**Review by JieDing — APPROVED** [review]
LGTM

**Pull request merged**